docs/examples/metrics/example.py

```diff
@@ -1,16 +1,16 @@
 from typing import Iterable
 
-from opentelemetry._metrics import (
+from opentelemetry.exporter.otlp.proto.grpc._metric_exporter import (
+    OTLPMetricExporter,
+)
+from opentelemetry.metrics import (
     Observation,
     get_meter_provider,
     set_meter_provider,
 )
-from opentelemetry._metrics._internal.instrument import CallbackOptions
-from opentelemetry.exporter.otlp.proto.grpc._metric_exporter import (
-    OTLPMetricExporter,
-)
-from opentelemetry.sdk._metrics import MeterProvider
-from opentelemetry.sdk._metrics.export import PeriodicExportingMetricReader
+from opentelemetry.metrics._internal.instrument import CallbackOptions
```

Comment:
Can you not directly import `CallbackOptions` from `opentelemetry.metrics`? I believe we don't expect users to access any private stuff

Reply:
Fixed